pkg/ddevapp/global_dotddev_assets/commands/web/wp

```diff
@@ -6,4 +6,7 @@
 ## ProjectTypes: wordpress
 ## ExecRaw: true
 
+# Ignore anything we find in the mounted global commands
+PATH=${PATH//\/mnt\/ddev-global-cache\/global-commands\/web/}
+
```

Comment:
I didn't check at first, but there's `/usr/local/bin/wp` available all the time, so it's most likely not needed here.

Reply:
Agreed. I did it anyway based on it being in your list. Does no harm.